stories/features/newTab/components/featureBlock/style.ts

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+import { FeatureBlockProps } from './index'
+
+export const StyledFeatureBlock = styled<{}, 'div'>('div')`
```

Comment:
Nit: More readable to specify Prop types in component declaration here, instead of in the inline functions below (repeated)

Reply:
agree, done